dashboard/test/ui/test_status.js

```diff
@@ -171,7 +172,7 @@ function testFromS3Key(key) {
   // unlikely.
   var test = tests[browser][feature];
   while (!test && feature.indexOf('_') >= 0) {
-    feature = feature.replace('_', '/'); // Replaces first underscore only.
+    feature = feature.replace(/_/, '/'); // Replaces first underscore only.
```

Comment:
This comment was a lie (and so is the documentation on MDN apparently) - when using a `string` needle the replacement was global even though I intended it to be only the first occurrence.  Using a `RegExp` needle it should be just the first one, since I'd have to pass the `/g` global flag to replace all.